src/psyclone/tests/gocean1p0_test.py

```diff
@@ -403,27 +411,27 @@ def test_scalar_float_arg_from_module():
         "    USE kind_params_mod\n"
         "    IMPLICIT NONE\n"
         "    CONTAINS\n"
-        "    SUBROUTINE invoke_0_bc_ssh(a_scalar, ssh_fld)\n"
-        "      USE kernel_scalar_float, ONLY: bc_ssh_code\n"
+        "    SUBROUTINE invoke_0_bc_ssh(ssh_fld)\n"
```

Comment:
Was this a bug/just wrong before?

Reply:
Yes, I think the imported symbol was added with the name clash (double declaration with the argument symbol) and the gen_code wasn't complaining before.

Now this name clash is not allowed anymore and I needed to fix it (remove it from the argument list).